packages/preview/fractusist/0.1.0/examples/dragon-curve-n12.typ

```diff
@@ -0,0 +1,18 @@
+#set document(date: none)
+
+
+#import "/src/lib.typ": *

```

Comment:
I think this example should use the package import.
```suggestion
#import "@preview/fractusist:0.1.0": *
```

Reply:
I've modified it as suggested.